src/main/java/org/llorllale/cactoos/matchers/Throws.java

```diff
@@ -66,35 +82,76 @@
      * @param msg The expected exception message.
      * @param type The expected exception type.
      */
-    public Throws(final String msg, final Class<? extends Exception> type) {
+    public Throws(
+        final String msg, final Class<? extends Exception> type
+    ) {
+        this(
+            type,
+            actual -> actual.equals(msg),
+            new UncheckedText(
+                new FormattedText("message '%s'", msg)
+            ).asString()
+        );
+    }
+
+    /**
+     * Ctor.
+     * @param type The expected exception type.
+     * @param expectation The expectation which the exception message must
+     *  match.
+     */
+    public Throws(
+        final Class<? extends Exception> type,
```

Comment:
the same, type vs type

Reply:
@ilyakharlamov Same reply.

@ Whoever will be doing quality assurance - I know one of the requirements for good quality is that every comment has to be addressed. Does it apply also in cases like this, where one reply covers several comments?